styleguide/_layouts/default.jade

```diff
@@ -31,50 +31,48 @@ html(lang="en")
               li.navZoolander-item
-                 a(href="global/").navZoolander-dropdownLink Overview
+                a(href="global/").navZoolander-dropdownLink Overview
               li.navZoolander-item
-                 a(href="global/brand-elements").navZoolander-dropdownLink Brand Elements
-
-          li.navZoolander-item
-            a(href="global/voice").navZoolander-link Voice and Tone
-
-          li.navZoolander-item
-            a(href="global/components").navZoolander-link Components
+                a(href="global/brand-elements").navZoolander-dropdownLink Brand Elements
+              li.navZoolander-item
+                a(href="global/voice").navZoolander-dropdownLink Voice and Tone
 
           li.navZoolander-item
-            a(href="#").navZoolander-link.navZoolander-hasDropdown Assets/Elements
+            a(href="#").navZoolander-link.navZoolander-hasDropdown Zoolander Global Elements
             ul.navZoolander-dropdown
               li.navZoolander-dropdownItem
                 a(href="derek/assets/").navZoolander-dropdownLink Overview
               li.navZoolander-dropdownItem
-                a(href="derek/assets/icon-font").navZoolander-dropdownLink Icon Font
-              li.navZoolander-dropdownItem
-                a(href="derek/assets/forms").navZoolander-dropdownLink Forms
-              li.navZoolander-dropdownItem
-                a(href="derek/assets/buttons").navZoolander-dropdownLink Buttons/CTAs
-              li.navZoolander-dropdownItem
                 a(href="derek/assets/typography").navZoolander-dropdownLink Typography
               li.navZoolander-dropdownItem
                 a(href="derek/assets/photography").navZoolander-dropdownLink Photography/Banners
               li.navZoolander-dropdownItem
-                a(href="derek/assets/icon-font").navZoolander-dropdownLink Tabs
+                a(href="derek/assets/swatches").navZoolander-dropdownLink Colors/Swatches
+              li.navZoolander-dropdownItem
+                a(href="derek/assets/icon-font").navZoolander-dropdownLink Icon Font
+              li.navZoolander-dropdownItem
+                a(href="derek/assets/buttons").navZoolander-dropdownLink Buttons/CTAs
               li.navZoolander-dropdownItem
                 a(href="derek/examples/tables").navZoolander-dropdownLink Tables
+              li.navZoolander-dropdownItem
+                a(href="derek/assets/forms").navZoolander-dropdownLink Forms
+
+          //Add in themes dropdown when there are multiple
 
           li.navZoolander-item
             a(href="#").navZoolander-link.navZoolander-hasDropdown Derek Theme
             ul.navZoolander-dropdown
               li.navZoolander-item
                 a(href="derek/").navZoolander-dropdownLink Overview
               li.navZoolander-item
-                a(href="derek/examples").navZoolander-dropdownLink Examples
+                a(href="derek/solutions").navZoolander-dropdownLink Solutions
```

Comment:
By moving everything out of /examples/ we no longer have the auto-collapse navigation feature.  Please adjust the JS accordingly. 

https://github.com/rackerlabs/zoolander/blob/bcdb78584d9a2a614c13e33b773846f0880cf38e/styleguide/js/zoolander.js#L184